src/scipp/core/bins.py

```diff
@@ -394,7 +424,12 @@ def histogram(x: Union[_cpp.DataArray, _cpp.Dataset], *,
     scipp.bin:
         For binning data.
     """
-    return _call_cpp_func(_cpp.histogram, x, bins)
+    if edges is not None:
+        kwargs = dict(**edges, **kwargs)
```

Comment:
`kwargs` overrides `edges`. Is this intentional? Or should we raise on clashes?

Reply:
This syntax *does* raise?